economics/__init__.py

```diff
@@ -1,2 +1,2 @@
-from cpi import CPI
-from inflation import Inflation
+from economics.cpi import CPI
```

Comment:
Relative imports would also have worked but since we don't have lots and lots of subpackages this is fine :+1: